src/shims/windows/handle.rs

```diff
@@ -16,6 +17,8 @@ pub enum Handle {
     Null,
     Pseudo(PseudoHandle),
     Thread(ThreadId),
+    File(i32),
```

Comment:
Are file IDs truly *signed*?

Reply:
Not really, it just made the types line up better.

Reply:
I see several casts between `i32` and `u32` in this PR. In which sense do the types line up better with `i32`?

Reply:
There were a lot more without it - most uses pass the contained number to FdMap, which expects `i32`, while this just needs it at the Handle/Scalar boundary. I can mock up the diff for that change when I get a chance, if you'd like to see the difference (none also that with that model, all new shims that deal with files will also be adding casts, but this way they mostly don't)

Reply:
Ah, they are, true.

We should probably have at least a type alias to make the intent more clear, like
```rust
type FdNum = i32;
```
(A newtype might be better but is a lot more work so let's hold off on that.)